articles/cosmos-db/sql/tutorial-bicep-deploy-sql-todoapp-on-aks.md

```diff
@@ -0,0 +1,297 @@
+
+    :::image type="content" source="./media/tutorial-aks-bicep-sql-todo/build_image.png" alt-text="Build Image VS Code":::
+
+2. To push the built image to ACR open the Docker tab. You'll find the built image under the Images node. Open the todo node, right-click on latest and select "Push...". 
+3. You'll then get prompts to select your Azure Subscription, ACR, and Image tag. Image tag format should be {acrname}.azurecr.io/todo:latest.
+
+    :::image type="content" source="./media/tutorial-aks-bicep-sql-todo/image_push.png" alt-text="Push Image to ACR":::
+
+4. Wait for VS Code  to push the  image to ACR.
+
+
+## Prepare Deployment YAML
+
+Using the following YAML template create a akstododeploy.yml file. Make sure to replace the values for {ACR Name}, {Image Name}, {Version}, and {Resource Group Name} placeholders.
+
+```yml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: todo
+  labels:
+    aadpodidbinding: "cosmostodo-apppodidentity"
+    app: todo
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: todo
+  template:
+    metadata:
+      labels:
+        app: todo
+        aadpodidbinding: "cosmostodo-apppodidentity"
+    spec:
+      containers:
+      - name: mycontainer
+        image: "{ACR Name}/{Image Name}:{Version}"   # update as per your environment, example myacrname.azurecr.io/todo:latest. Do NOT add https:// in ACR Name
+        ports:
+        - containerPort: 80
+        env:
+        - name: KeyVaultName
+          value: "{Resource Group Name}kv"       # Replace resource group name. Key Vault name is generated by Bicep
+      nodeSelector:
+        kubernetes.io/os: linux
+      volumes:
+        - name: secrets-store01-inline
+          csi:
+            driver: secrets-store.csi.k8s.io
+            readOnly: true
+            volumeAttributes:
+              secretProviderClass: "azure-kvname-podid"       
+---
+    
+kind: Service
+apiVersion: v1
+metadata:
+  name: todo
+spec:
+  selector:
+    app: todo
+    aadpodidbinding: "cosmostodo-apppodidentity"    
+  type: LoadBalancer
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 80
+``` 
+
+## Apply Deployment YAML
+
+The following command deploys the application pods and exposes the pods via a load balancer.
+
+```azurecli-interactive
+kubectl apply -f akstododeploy.yml --namespace 'my-app'
+```
+
+## Test the application
+
+When the application runs, a Kubernetes service exposes the application front end to the internet. This process can take a few minutes to complete
+
+Run the following command to view the external IP exposed by the load balancer
+
+```azurecli-interactive
+kubectl get services --namespace "my-app"
+```
+
+Open the IP received as output in a browser to access the application.
+```
+
+## Clean up the resources
+
+To avoid Azure charges, you should clean up unneeded resources.  When the cluster is no longer needed, use the below commands to delete the Resource Group and Deployment
```

Comment:
_not needed resources._

Reply:
This sentence was copied from an existing tutorial. Hence not changing.